### PR TITLE
Redirect to  accuratte error page : "account do not exist" (In Progress) 

### LINKED ISSUE #230: Confirmation screen after password reset - account do not exist
**To Reproduce**
1. Go to the page: https://dev.devadventcalendar.pl/Account/Login
2. Click on 'I forgot the password'
3. Type email address which do not exist in DB and click button 'reset'
4. Below confirmation screen is visible:
image

**Current behavior**
Information can point that this email address exist, and operation of reset password was successful.

**Expected behavior**
After clicking 'reset' option for email address not existing in DB there should be information that this account do not exist and option of reset password for non-existent account is not possible

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by Oliweirdo**
## Description

Added new View model, and redirect to error page.

## Where should the reviewer start?

Go to the page: https://dev.devadventcalendar.pl/Account/Login 
and Click on 'I forgot the password'

## Motivation and context

closes #230

## How has this been tested?

After clicking 'reset' option for email address not existing in DB there should be information that this account do not exist and option of reset password for non-existent account is not possible

## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by Oliweirdo** [comment]
Finally, can`t find the error page, I`m working on it.

**Comment by WTobor** [comment]
@Oliweirdo what about this PR? Is it still needed or should we delete it?

**Comment by Oliweirdo** [comment]
I added some changes according to  @smiechuskce  suggestions. I think is done and we can close it.

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Properties/launchSettings.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
       "launchUrl": "https://localhost:44301/",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
-      }
+      },
+      "ancmHostingModel": "InProcess"
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Properties/launchSettings.json]: this change is out of scope

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/DevAdventCalendarCompetition.csproj [hunk 2]
```diff
@@ -33,9 +33,14 @@
...
     </PackageReference>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.2.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="2.2.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.2.0" />
     <PackageReference Include="Microsoft.VisualStudio.Web.BrowserLink" Version="2.2.0" />
-    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.3" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.4" PrivateAssets="All" />
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/DevAdventCalendarCompetition.csproj]: these changes are out of scope

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Properties/launchSettings.json (hunk 1 above)
Comment (WTobor) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Properties/launchSettings.json]: @Oliweirdo remember to resolve conversation after changes :)

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/DevAdventCalendarCompetition.csproj (hunk 2 above)
Comment (WTobor) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/DevAdventCalendarCompetition.csproj]: @Oliweirdo remember to resolve conversation after changes :)
